client/js/libs/handlebars/friendlydate.js

```diff
@@ -4,10 +4,10 @@ const moment = require("moment");
 
 module.exports = function(time) {
 	// See http://momentjs.com/docs/#/displaying/calendar-time/
-	return moment(new Date(time)).calendar(null, {
+	return moment(time).calendar(null, {
 		sameDay: "[Today]",
 		lastDay: "[Yesterday]",
-		lastWeek: "L", // Locale
-		sameElse: "L"
+		lastWeek: "MMMM DD, YYYY",
+		sameElse: "MMMM DD, YYYY"
```

Comment:
I think locale formatting is not that good, especially when we can't specify that we want the longer representation. The format I chose here is *not* american, but rather a sane approach to formatting dates, where full month name is displayed. This removes any confusion that there can be for "is it month or day?"